### ISSUE TITLE
Verge Mining with Lyra2REv2(XVG) - Any pools support this?

### ISSUE THREAD

**Issue by pokepud3** [Sniffdog/Sniffdogminer#61]
This has the been the top 5 coins to mine for a while on whattomine.com, just curious if we'll see any support for it?

**Comment by Sniffdog** [comment]
yes I did place excavator in with Sniffdog.... hashes a little better than Klaust
